### PR TITLE
Task/MYR-518, 507, 522

### PR DESCRIPTION
**Pull request by ical10**

### CODE REVIEW

File: src/helpers/transformTokenValue.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import {Transaction} from 'src/interfaces/transaction';
 enum TokenID {
```
Comment (irmannmal) [inline src/helpers/transformTokenValue.ts:3]: we need to discuss this helper, i think this helper can be simplify, and make this helper dynamic token transform

File: .gitignore [hunk 2]
```diff
@@ -86,3 +86,4 @@ prisma
 # next.js
 .next
 /public/sw-env.js
+.env.local
```
Comment (irmannmal) [inline .gitignore:89]: we don't expect this, `.env` is a context

**Review by irmannmal — CHANGES_REQUESTED** [review]

File: src/helpers/transformTokenValue.ts (hunk 1 above)
Comment (ical10) [reply src/helpers/transformTokenValue.ts:3]: Okay, let's discuss this on discord @irmannmal

File: .gitignore (hunk 2 above)
Comment (ical10) [reply .gitignore:89]: removed, done

**Review by irmannmal — APPROVED** [review]

**Pull request merged**